pkg/guestagent/hwclock/hwclock_linux.go

```diff
@@ -0,0 +1,23 @@
+package hwclock
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+func Get() (t time.Time, err error) {
+	cmd := exec.Command("hwclock")
+	cmd.Stdout = &bytes.Buffer{}
+	err = cmd.Run()
+	if err != nil {
+		return
+	}
+	t, err = time.Parse("2006-01-02 15:04:05.999999-07:00", strings.TrimSpace(cmd.Stdout.(*bytes.Buffer).String()))
+	return
+}
+
+func HcToSys() error {
+	return exec.Command("bash", "-c", "hwclock --hctosys").Run()
```

Comment:
No need to use bash

Reply:
Unfortunately, bash is required here. hwclock doesn't reset system clock without environment set.

Reply:
What environment is needed?
Setting `cmd.Env` should work?